compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/Symbols.java

```diff
@@ -222,6 +222,21 @@ public static BXMLNSSymbol createXMLNSSymbol(Name name,
         return new BXMLNSSymbol(name, nsURI, pkgID, owner, pos, origin);
     }
 
+    public static BVarSymbol createBVarSymbol(long flags,
+                                              Name name,
+                                              PackageID pkgID,
+                                              BType type,
+                                              BSymbol owner,
+                                              Location pos,
+                                              SymbolOrigin origin) {
+        return new BVarSymbol(flags, name, pkgID, type, owner, pos, origin);
+    }
+
+    public static BVarSymbol createBVarSymbolForType(BType type) {
```

Comment:
Let's make this more specific since it is only used with tuple members atm.

```suggestion
    public static BVarSymbol createVarSymbolForTupleMember(BType memberType) {
```